GameServerLib/GameObjects/Spells/Spell.cs

```diff
@@ -231,17 +238,23 @@ public void AddProjectile(string nameMissile, float fromX, float fromY, float to
                 SpellData.MissileSpeed,
                 nameMissile,
                 SpellData.Flags,
-                FutureProjNetId
+                projNetId,
+                isServerOnly
             );
+            if (p != null)
```

Comment:
Why `p` will be `null`?

Reply:
And this is a good point to use `?.`

Reply:
Only did it out of instinct, didn't have any cases yet where it was null.

Reply:
Removed the check.